xdsl/dialects/riscv.py

```diff
@@ -704,6 +725,22 @@ class LiOp(RdImmOperation):
     name = "riscv.li"
 
 
+@irdl_op_definition
+class EcallOp(RdRsOperation):
```

Comment:
Should `ecall` have `rd` and `rs`? I think it should be a `NullaryOperation`, right?